### PR TITLE
[ASM] Add ASRs EXPANDR-11245

### PR DESCRIPTION
**Pull request by johnnywilkes**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
https://jira-dc.paloaltonetworks.com/browse/EXPANDR-11245

## Description
Added the following Attack Surface Rules to Automated Remediation Requirements: 
- TFTP Server
- Libssh
- Insecure Bitvise SSH Server
- Insecure SFTPGo

## Must have
- [ ] Tests
- [X] Documentation

### CODE REVIEW

**Comment by johnnywilkes** [comment]
@ShirleyDenkberg , please review

**Comment by johnnywilkes** [comment]
@BigEasyJ / @capanw , please review

File: Packs/CortexAttackSurfaceManagement/pack_metadata.json [hunk 1]
```diff
@@ -2,7 +2,7 @@
     "name": "Cortex Attack Surface Management",
     "description": "Content for working with Attack Surface Management (ASM).",
     "support": "xsoar",
-    "currentVersion": "1.7.48",
+    "currentVersion": "1.7.49",
```
Comment (BigEasyJ) [inline Packs/CortexAttackSurfaceManagement/pack_metadata.json:5]:
```suggestion
    "currentVersion": "1.7.51",
```

**Review by BigEasyJ — APPROVED** [review]

**Comment by johnnywilkes** [comment]
@israelpoli / @tomer-pan , this has been approved by Xpanse, please merge

**Comment by ShirleyDenkberg** [comment]
@BigEasyJ @israelpoli @tomer-pan Doc review completed.

File: Packs/CortexAttackSurfaceManagement/pack_metadata.json (hunk 1 above)
Comment (israelpoli) [reply Packs/CortexAttackSurfaceManagement/pack_metadata.json:5]: The comment is probably a mistake
the current version is 1.7.48
I continue with the merger

**Review by israelpoli — APPROVED** [review]
💪

**Pull request merged**